agents/sftp-to-gcs/deploy/init-containers/transporter-sftp-to-gcs-agent/application.conf.ctmpl

```diff
@@ -22,9 +22,11 @@ org.broadinstitute.transporter {
     }
 
     runner-config {
+        mib-per-step = {{env "MIB_PER_TRANSFER_STEP"}}
+
         sftp {
             host = "{{env "SFTP_HOST"}}"
-            port = "{{env "SFTP_PORT"}}"
+            port = {{env "SFTP_PORT"}}
```

Comment:
is this not in quotes because the port number is not a string?

Reply:
Yup